src/packs.c

```diff
@@ -111,11 +110,15 @@ int download_subscribed_packs(struct list *subs, bool required)
 		iter = iter->next;
 
 		if (sub->oldversion == sub->version) { // pack didn't change in this release
+			fprintf(stderr, "no changes for %s pack for current version (pack %d of %d)\n",
+				sub->component, ++complete, list_length);
```

Comment:
I'm not convinced we should print a progress message for this case. If a user has many bundles installed and is updating all the time, not many bundles will change (on average) during an update, but all of these extra progress messages will be printed.